src/makeRobotAccountant.js

```diff
@@ -6,7 +6,27 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callsLeft = 3;
+  let isEvenCall = true;
+
+  return (firstNum) => {
+    const sumFunc = (secondNum) => firstNum + secondNum;
+
+    if (callsLeft) {
+      callsLeft -= 1;
+
+      return (secondNum) => sumFunc(secondNum);
+    }
+
+    if (isEvenCall) {
+      isEvenCall = false;
+
+      return () => 'Bzzz... Error!';
+    }
+    isEvenCall = true;
+
+    return (secondNum) => sumFunc(secondNum);
+  };
```

Comment:
A lot of redundant logic. Try to use only one varibale which will count the number of calls and use this condition
```
 if (count > 3 && count % 2 === 0) {
        return 'Bzzz... Error!';
      }
```